sklearn/cluster/hierarchical.py

```diff
@@ -229,7 +231,7 @@ def ward_tree(X, connectivity=None, n_clusters=None, return_distance=False):
                           stacklevel=2)
         X = np.require(X, requirements="W")
         out = hierarchy.ward(X)
-        children_ = out[:, :2].astype(np.intp)
+        children_ = out[:, :2].astype(np.intp, copy=False)
```

Comment:
Whether or not the argument about view not being released applies here or not depends on whether we have return_distance or not. I am not sure whether or not the copy should be kept or not.

This comment does not ask for a change in the diff. It's just a reflection as I pass by.

Reply:
Removed `copy=False` here to use the defaults when in doubt.